CMakeLists.txt

```diff
@@ -138,15 +144,84 @@ find_library(FREETYPE freetype)
 find_library(GEOTIFF geotiff)
 find_library(JPEG jpeg)
 
-if (BUILD_GUI)
-	find_package (OpenGL REQUIRED)
-	include_directories (SYSTEM ${OPENGL_INCLUDE_DIRS})
-	if (WIN32)
-		find_library (GLEW glew32)
-	else ()
-		find_library (GLEW GLEW)
-	endif ()
-endif ()
+# find_package(Python)
+#   Output
+#       ${Python_VERSION}
+#       ${Python_LIBRARIES}
+#       ${Python_INCLUDE_DIRS}
+#       ${Python_SITELIB}
+#       ${Python_NumPy_VERSION}
+#       ${Python_NumPy_INCLUDE_DIRS}
+#   https://cmake.org/cmake/help/v3.12/module/FindPython.html
+
+function(FIND_BUNDLED_PYTHON)
+    # Find_Python supports Python_ROOT_DIR however vapor's bundled python distribution
+    # does not conform to its requirements so this manually configures the results
```

Comment:
Are "Find_Python" and "Python_ROOT_DIR" mis-capitalized in this comment?  I'm unable to find what they're referring to in our repository, or in online cmake documentation.

Reply:
https://cmake.org/cmake/help/v3.14/module/FindPython.html